data-service/src/main/java/uk/gov/gchq/palisade/service/data/stream/SerDesConfig.java

```diff
@@ -0,0 +1,130 @@
+import org.apache.kafka.common.serialization.Deserializer;
+import org.apache.kafka.common.serialization.Serializer;
+import org.apache.kafka.common.serialization.StringSerializer;
+import org.springframework.core.serializer.support.SerializationFailedException;
+
+import uk.gov.gchq.palisade.service.data.model.AuditErrorMessage;
+import uk.gov.gchq.palisade.service.data.model.AuditSuccessMessage;
+import uk.gov.gchq.palisade.service.data.model.DataRequest;
+
+import java.io.IOException;
+import java.nio.charset.Charset;
+
+
+/**
+ * Static configuration for kafka key/value serialisers
+ * - Each output has a pair of key/value serialisers
+ * In general, the keys are not used so the choice of serialiser is not important
+ */
+public final class SerDesConfig {
+    private static final ObjectMapper MAPPER = new ObjectMapper();
+    private static final String SERIALIZATION_FAILED_MESSAGE = "Failed to serialize ";
+    private static final String DESERIALIZATION_FAILED_MESSAGE = "Failed to deserialize ";
+
+    private SerDesConfig() {
+        // Static collection of objects, class should never be instantiated
+    }
+
+
+    /**
+     * Kafka key serialiser for downstream messages going out as output
+     *
+     * @return an appropriate key serialiser for the topic's message content
+     */
+    public static Serializer<String> requestKeySerializer() {
+        return new StringSerializer();
+    }
+
+    /**
+     * Kafka value serialiser for downstream messages going out as output
+     *
+     * @return an appropriate value serialiser for the topic's message content (AttributeMaskingResponse)
+     */
+    public static Serializer<byte[]> passthroughValueSerializer() {
+        return (String ignored, byte[] bytes) -> bytes;
+    }
+
+    /**
+     * Kafka key serialiser for downstream messages going out as success
+     *
+     * @return an appropriate key serialiser for the topic's message content
+     */
+    public static Serializer<String> successKeySerializer() {
+        return new StringSerializer();
+    }
+
+    /**
+     * Kafka value serialiser for downstream messages going out as success
+     *
+     * @return an appropriate value serialiser for the topic's message content (AuditMessage)
+     */
+    public static Serializer<AuditSuccessMessage> successValueSerializer() {
+        return (String ignored, AuditSuccessMessage auditSuccessMessage) -> {
+            try {
+                return MAPPER.writeValueAsBytes(auditSuccessMessage);
+            } catch (JsonProcessingException e) {
+                throw new SerializationFailedException(SERIALIZATION_FAILED_MESSAGE + auditSuccessMessage.toString(), e);
+            }
+        };
+    }
+
+    /**
+     * Kafka value deserialiser for upstream messages coming in as input
+     *
+     * @return an appropriate value deserialiser for the topic's message content (PolicyRequest)
+     */
+    public static Deserializer<DataRequest> requestValueDeserializer() {
```

Comment:
Having this next to the `requestKeySerializer` would make it easier on the eye.

Reply:
actually what you found was some dead code I'm sure that I had already deleted and it looks like it has come back to haunt me.  Deleted all of it again